### PR TITLE
Static library compatible

### PR DESCRIPTION
**Pull request by snowie2000**

### CODE REVIEW

**Comment by rwfpl** [comment]
I've briefly looked at your patch and I'll need to make few adjustments before merge. Thanks for contribution.

File: src/wow64ext.cpp [hunk 1]
```diff
@@ -595,3 +604,33 @@ extern "C" __declspec(dllexport) BOOL __cdecl SetThreadContext64(HANDLE hThread,
...
+	if (0 == ldrLoadDll)
+	{
+		ldrLoadDll = GetProcAddress64(getNTDLL64(), "LdrLoadDll");
+		if (0 == ldrLoadDll)
+			return 0;
+	}
+	UNICODE_STRING szDll = { 0 };
+	szDll.Buffer = (DWORD64)lpLibFileName;
+	szDll.Length = wcslen(lpLibFileName)*sizeof(WCHAR);
+	szDll.MaximumLength = szDll.Length;
```
Comment (justanotheranonymoususer) [inline src/wow64ext.cpp]: Should be `szDll.Length + 1`

File: src/wow64ext.cpp [hunk 2]
```diff
@@ -595,3 +604,33 @@ extern "C" __declspec(dllexport) BOOL __cdecl SetThreadContext64(HANDLE hThread,
...
+WOW_EXPORT DWORD64 __cdecl LoadLibraryW64(LPCWSTR lpLibFileName)
+{
+	static DWORD64 ldrLoadDll = 0;
+	if (0 == ldrLoadDll)
+	{
+		ldrLoadDll = GetProcAddress64(getNTDLL64(), "LdrLoadDll");
+		if (0 == ldrLoadDll)
+			return 0;
+	}
+	UNICODE_STRING szDll = { 0 };
```
Comment (justanotheranonymoususer) [inline src/wow64ext.cpp]: Can use `_UNICODE_STRING_T<DWORD64>`

File: src/wow64ext.cpp (hunk 1 above)
Comment (justanotheranonymoususer) [inline src/wow64ext.cpp]: `szDll.Length + sizeof(WCHAR)` actually

**Comment by rwfpl** [comment]
obsolete

**Pull request closed without merging**